_data/events1.yml

```
@@ -0,0 +1,78 @@
+- category: Online Event
+  category_name: Online Event
+  date: Tue, Aug 13, 2024
+  description: "Kedro is an open-source Python framework to create reproducible, maintainable,\
+    \ and modular data science code. It uses software engineering best practices to\
+    \ help you build production-ready data science pipelines.\nSchedule:\n\u23F0 19:00\
+    \ - 19:15 Intro to Women Coding Community\n\u23F0 19:15 - 19:45: Talk by Merel\
+    \ Theisen\n\u23F0 19:45 - 20:00: Q&A"
+  expiration: '20240813'
+  host: Silke Nodwell and Irina Kamalova
+  image:
+    alt: 'Kedro: An open-source project for building maintainable data science pipelines
+      cover photo'
+    path: https://secure.meetupstatic.com/photos/event/6/4/c/e/event_522085806.webp
+  link:
+    path: https://meetup.com/women-coding-community/events/301960944/?eventOrigin=group_events_list
+    target: _target
+    title: 'Kedro: An open-source project for building maintainable data science pipelines'
+  speaker: Merel Theisen
+  time: 6:00 PM UTC
+  title: 'Kedro: An open-source project for building maintainable data science pipelines'
+- category: Online Event
+  category_name: Online Event
+  date: Wed, Aug 21, 2024
+  description: "In this final talk of the series on Serverless Computing, we will\

```

Ideally, here you would do like this

```suggestion
  description: | 
          In this final talk of the series on Serverless Computing, 
          we will focus on Edge Computing, understanding AWS Lambda@Edge and Amazon CloudFront
... 
```